ui/src/app/components/help-sidebar.tsx

```diff
@@ -539,49 +552,109 @@ export const HelpSidebar = fp.flow(
+              color: colors.asyncOperationStatus.starting,
             }}/>
           }
           {status === RuntimeStatus.Stopped
             && <FontAwesomeIcon icon={faCircle} style={{
               ...styles.runtimeStatusIcon,
               ...styles.runtimeStatusIconOutline,
-              color: colors.runtimeStatus.stopped,
+              color: colors.asyncOperationStatus.stopped,
             }}/>
           }
           {status === RuntimeStatus.Running
             && <FontAwesomeIcon icon={faCircle} style={{
               ...styles.runtimeStatusIcon,
               ...styles.runtimeStatusIconOutline,
-              color: colors.runtimeStatus.running,
+              color: colors.asyncOperationStatus.running,
             }}/>
           }
           {(status === RuntimeStatus.Stopping
           || status === RuntimeStatus.Deleting)
             && <FontAwesomeIcon icon={faSyncAlt} style={{
               ...styles.runtimeStatusIcon,
               ...styles.rotate,
-              color: colors.runtimeStatus.stopping,
+              color: colors.asyncOperationStatus.stopping,
             }}/>
           }
           {status === RuntimeStatus.Error
             && <FontAwesomeIcon icon={faCircle} style={{
               ...styles.runtimeStatusIcon,
               ...styles.runtimeStatusIconOutline,
-              color: colors.runtimeStatus.error,
+              color: colors.asyncOperationStatus.error,
             }}/>
           }
         </FlexRow>
       </FlexRow>;
     }
 
+    withinPastTwentyFourHours(epoch) {
+      const completionTimeMoment = moment(epoch);
+      const twentyFourHoursAgo = moment().subtract(1, 'days');
+      return completionTimeMoment.isAfter(twentyFourHoursAgo);
+    }
+
+    displayExtractionIcon(icon: IconConfig) {
+      const {extractionJobs} = this.state;
+      const jobsByStatus = fp.groupBy('status', extractionJobs);
+      let status;
+      // If any jobs are currently active, show the 'sync' icon corresponding to their status.
+      if (jobsByStatus[TerraJobStatus.RUNNING]) {
+        status = TerraJobStatus.RUNNING;
+      } else if (jobsByStatus[TerraJobStatus.ABORTING]) {
+        status = TerraJobStatus.ABORTING;
+      } else if (jobsByStatus[TerraJobStatus.SUCCEEDED] || jobsByStatus[TerraJobStatus.FAILED]) {
+        // Otherwise, show the status of the most recent completed job, if it was completed within the past 24h.
+        const completedJobs = fp.concat(jobsByStatus[TerraJobStatus.SUCCEEDED] || [], jobsByStatus[TerraJobStatus.FAILED] || []);
+        const recentCompletedJobs = fp.flow(
+          // This could be phrased as fp.sortBy('completionTime') but it confuses the compile time type checker
+          fp.sortBy((job: GenomicExtractionJob) => job.completionTime),
+          fp.reverse
+        )(completedJobs);
+        if (!fp.isEmpty(recentCompletedJobs) && this.withinPastTwentyFourHours(recentCompletedJobs[0].completionTime)) {
```

Comment:
opt nit: this 24h check could instead be a filter within the flow, might be slightly more readable IMO

If you did that, you could also end the flow with `fp.head`, then make this if check very simple: `if (mostRecentCompletedJob) { ...`

Reply:
I thought about that, but I figured why check every job for recency when we could just check the first one. It's likely not going to be thousands of jobs, though, so readability is probably the way to go here.